build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -9,3 +9,7 @@ flowCollectorAddr: ""
 # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 # Defaults to "60s".
 flowExportInterval: 60s
+
+# Provide the protocol for flow aggregator collecting process, which is tcp or udp.
+# Defaults to "udp"
+collectingProtocol: "udp"
```

Comment:
use the name `flowCollectorProtocol` to be consistent with the code.

Reply:
I chose this variable name because I don’t want others to misunderstand it as an external collector’s protocol, which is actually the protocol used by flow aggregator collecting process.

Reply:
Good point.
However, I propose following for more clarity. 
`flowCollectorAddr` -> `externalFlowCollectorAddr`
`collectingProtocol` -> `aggregatorFlowCollectorProtocol` or `aggregatorCollectorProtocol` in short.

What I have seen with configMap parameters with Flow Exporter, many times folks overlook the description, so names are better to be more clearer even though slightly descriptive IMO.

Reply:
Thanks, changed it to externalFlowCollectorAddr and aggregatorFlowCollectorProtocol. Also change the default protocol of aggregatorFlowCollectorProtocol into tcp.